sections/Design.tex

```diff
@@ -34,8 +34,76 @@ \subsection{Premade functions (rename)}
 
 
 \section{Control Flow}
-\subsection{For-loop}
+Control flow is the parts of the language which will make ``decitions''.
+Often these are separated into two primary groups: iteration and condition. 
+An iteration is a statement which gives the program the possibility to iterate over the same code a given number of times. 
+Common loops in programming languages are: do..while, while, for and foreach. 
+The iteration of a loop can be broken in our language, to jump out of the loop, by a statement often called \texttt{break}, and the ability to skip to start of the next iteration by the keyword \texttt{continue}.
 

```

Comment:
Der er ingen syntaks, og der er intet omkring for-loops her egentlig. Generelt er det bare om loops.